quinn-proto/src/connection/mod.rs

```diff
@@ -3161,6 +3357,18 @@ impl Connection {
         self.key_phase = !self.key_phase;
     }
 
+    fn peer_supports_ack_frequency(&self) -> bool {
+        self.peer_params.min_ack_delay.is_some()
+    }
+
+    /// Send an IMMEDIATE_ACK frame to the remote endpoint
+    ///
+    /// According to the spec, this will result in an error if the remote endpoint does not support
+    /// the Acknowledgement Frequency extension
+    pub(crate) fn immediate_ack(&mut self) {
```

Comment:
I'd say in this case the existence of the method is justified because it makes the code a bit more self-documenting: having a function called `immediate_ack` (plus docs explaining what it does) makes the code in my eyes easier to follow than writing `self.spaces[self.highest_space].immediate_ack_pending = true;`.

I can inline it, though, if you feel it really improves things.

Reply:
I mean, you can still add comments, right? I think putting this line in its own code block (surrounded by empty lines) with an explanatory comment makes the code just as well-documented, but still easier to follow due to less indirection.

I don't feel very strongly about it though, if you don't think it makes sense.

Reply:
I was about to inline the method, but then I realized it is also being used in tests, so IMO it makes sense to keep it after all